src/pages/Index/Events/index.tsx

```diff
@@ -67,7 +55,7 @@ export class Events extends Component {
                     </div>
 
                     <div className="content-events">
-                        <div className="events-slider">
+                        <div className="events-slider" ref={this.swiperRef}>
```

Comment:
```suggestion
                        <div className="events-slider">
```
除了这个初始化，其它地方完全没用上，这个变量全篇都删了吧。